### DISCUSSION TITLE
In deta base, last two digits are missing if int is included.

### DISCUSSION THREAD

**Discussion by ghost** [deta/cloud-docs#242] [Help] [answered]
The Python module stores a dictionary using put, but including an int in the dictionary would result in missing the last two digits.
Is this specification? The solution was to store all the ints converted to str.

Also, the module does not support asynchronous processing.
ASGIs such as FastAPI and Responder and web frameworks should use "run_in_executor"
Are there any plans to support asynchronous processing natively?

**Comment by jnsougata** [comment]
Asynchronous DETA BASE SDK is being developed. You can install and use alpha build.
Check docs here: https://docs.deta.sh/docs/base/async_sdk

**Comment by abdelhai** [comment, marked as answer]
hi, could you share your `put` statement? ints should always stay int up to 16 digits, otherwise, you should use a string: https://docs.deta.sh/docs/base/sdk#storing-numbers

this is a technical limitation of our database for now

as Sougata mentioned, we're working on a full-support for async, for now you can test the alpha

**Reply by ghost** [reply]
Thank you.
When I stored the Discord slowflake id, the last two digits were missing.
Process by string
I checked the operation of the α version, but it is working properly.
